### PR TITLE
[CI Visibility] - Add new section for BenchmarkDotnet support.

### PR DESCRIPTION
**Pull request by tonyredondo**
### What does this PR do?
Adds a new section to include the support of a new testing framework to CI Visibility.

### Motivation
We added support to a benchmark testing framework for .NET in CI Visibility.

### Preview

https://docs-staging.datadoghq.com/tony/ci-visibility-tests-benchmarkdotnet/continuous_integration/tests/dotnet

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/dotnet.md [hunk 1]
```diff
@@ -170,6 +173,51 @@ if (scope != null) {
...
+### Instrumenting BenchmarkDotNet tests
+
+To instrument your benchmark tests you need to:
+
+1. Add the [`Datadog.Trace.BenchmarkDotNet` NuGet package][12] to your project (eg. using `dotnet add package Datadog.Trace.BenchmarkDotNet`).
+2. Configure your project to use the `Datadog.Trace.BenchmarkDotNet` exporter using the `DatadogDiagnoser` attribute or the `WithDatadog()` extension method. For example:
+
+{{< tabs >}}
+
+{{% tab "Using the Attribute" %}}
```
Comment (albertvaka) [inline content/en/continuous_integration/tests/dotnet.md:185]:
```suggestion
{{% tab "Using the `[DatadogDiagnoser]` Attribute" %}}
```

**Review by albertvaka — APPROVED** [review]
LGTM

File: content/en/continuous_integration/tests/dotnet.md [hunk 2]
```diff
@@ -170,6 +173,51 @@ if (scope != null) {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
 
+### Instrumenting BenchmarkDotNet tests
+
+To instrument your benchmark tests you need to:
+
+1. Add the [`Datadog.Trace.BenchmarkDotNet` NuGet package][12] to your project (eg. using `dotnet add package Datadog.Trace.BenchmarkDotNet`).
```
Comment (estherk15) [inline content/en/continuous_integration/tests/dotnet.md:180]:
```suggestion
1. Add the [`Datadog.Trace.BenchmarkDotNet` NuGet package][12] to your project (for example, using `dotnet add package Datadog.Trace.BenchmarkDotNet`).
```

File: content/en/continuous_integration/tests/dotnet.md [hunk 3]
```diff
@@ -170,6 +173,51 @@ if (scope != null) {
...
+var config = DefaultConfig.Instance
+              .WithDatadog();
+
+BenchmarkRunner.Run<OperationBenchmark>(config);
+{{< /code-block >}}
+{{% /tab %}}
+
+{{< /tabs >}}
+
+3. Configure the reporting method as [explained above][13]
```
Comment (estherk15) [inline content/en/continuous_integration/tests/dotnet.md:218]:
```suggestion
3. [Configure the reporting method][13].
```

**Review by estherk15 — APPROVED** [review]
Just a few nitpicks, looks good!

**Comment by tonyredondo** [comment]
> Just a few nitpicks, looks good!

Thanks for the review.

**Pull request merged**